chemprop/cli/predict.py

```diff
@@ -252,27 +251,36 @@ def make_prediction_for_model(
     # else:
     #     cal_loader = None
 
-    logger.info(model)
+    individual_preds = []
+    for model_path in model_paths:
+        logger.info(f"Predicting with model at '{model_path}'")
 
-    trainer = pl.Trainer(
-        logger=False, enable_progress_bar=True, accelerator=args.accelerator, devices=args.devices
-    )
+        model = load_model(model_path, multicomponent)
 
-    predss = trainer.predict(model, test_loader)
+        logger.info(model)
 
-    # TODO: add uncertainty and calibration
-    # if cal_dset is not None:
-    #     if args.task_type == "regression":
-    #         model.loc, model.scale = float(scaler.mean_), float(scaler.scale_)
-    #     predss_cal = trainer.predict(model, cal_loader)[0]
+        trainer = pl.Trainer(
+            logger=False,
+            enable_progress_bar=True,
+            accelerator=args.accelerator,
+            devices=args.devices,
+        )
 
-    # TODO: might want to write a shared function for this as train.py might also want to do this.
-    df_test = pd.read_csv(args.test_path)
-    preds = torch.concat(predss, 0)
+        predss = trainer.predict(model, test_loader)
+
+        # TODO: add uncertainty and calibration
+        # if cal_dset is not None:
+        #     if args.task_type == "regression":
+        #         model.loc, model.scale = float(scaler.mean_), float(scaler.scale_)
+        #     predss_cal = trainer.predict(model, cal_loader)[0]
 
-    if isinstance(model.predictor, MulticlassClassificationFFN):
-        preds = torch.argmax(preds, dim=-1)
+        # TODO: might want to write a shared function for this as train.py might also want to do this.
+        preds = torch.concat(predss, 0)
+        if isinstance(model.predictor, MulticlassClassificationFFN):
+            preds = torch.argmax(preds, dim=-1)
+        individual_preds.append(preds)
 
+    average_preds = torch.mean(torch.stack(individual_preds).float(), dim=0)
```

Comment:
We should also compute and save the standard deviation?

Reply:
I would delegate this to the ensemble uncertainty methods.